Tests/Tests/CDTQIndexManagerEncryptionTests.m

```diff
@@ -0,0 +1,111 @@
+//
+//  CDTQIndexManagerEncryptionTests.m
+//  Tests
+//
+//  Created by Enrique de la Torre Fernandez on 08/04/2015.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "CloudantSyncTests.h"
+#import "CloudantTests+EncryptionTests.h"
+#import "CDTDatastoreManager+EncryptionKey.h"
+#import "CDTEncryptionKeyNilProvider.h"
+#import "FMDatabase+SQLCipher.h"
+
+#import "CDTQIndexManager.h"
+
+@interface CDTQIndexManagerEncryptionTests : CloudantSyncTests
```

Comment:
Worth noting here that these tests are checking that the encryption code works when the test app is compiled w/out SQLCipher support? I was confused as to why tests appeared repeated, a comment would've hopefully helped me avoid that mistake.